class_person.rb

```diff
@@ -1,27 +1,12 @@
-class Person
-  attr_accessor :name, :age
-  attr_reader :id
+class Person < Nameable
```

Comment:
- [ ] @BenMKT Great work on your decoration, however, you forgot to import the nameable class before the person class inherits from it. Also, your initialize method needs to accept two variables age and name, currently it's only accepting name.

Reply:
- [ ] [optional] @BenMKT I noted you removed the  `can_use_services?` and `of_age?` and modified the initialize method although they are not used in this milestone, they might be of help in future milestones, kindly consider restoring them.

Reply:
- Refactored Person class to resolve stated bugs